app/service/service-filter.js

```diff
@@ -1,25 +1,29 @@
+const { Logger } = require('@hmcts/nodejs-logging');
```

Comment:
No need for `{ }` if using `require`. You just need `const Logger = require(...)`